tools/cli/internal/cli/sunset/sunset.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sunset
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sunset",
+		Short: "Manage the Sunset API for the OpenAPI spec.",
+		Annotations: map[string]string{
+			"toc": "true",
+		},
```

Comment:
this is for generating docs, do we need ti here?
```suggestion
```

Reply:
no, I removed it